lib/mixlib/install/cli.rb

```diff
@@ -75,14 +75,15 @@ def download(product_name)
         if options[:url]
           say artifact.url
         else
+          url = artifact.is_a?(Array) ? artifact.map(&:url).first : artifact.url
```

Comment:
@wrightp : It's returning an array of hash for windows2008. Hence, I have to verify if the artifact instance is an array or a hash.